templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Description
+
+What I did and maybe screenshots.
+
+### General
+
+- [ ] I reviewed my code.
+- [ ] This PR is relatable to a ticket number (via branch name, title or description).
+- [ ] I updated the ticket on Redmine.
+- [ ] *This PR is deployed on the testing environment*
```

Comment:
I'd leave this point away.

Reply:
Why? I think it's useful to know it has been deployed on testing for the reviewer.

Reply:
It's just not the normal case.

Reply:
i'd say it can stay as optional point which i leave unticked if i didn't deploy it